fre/pp/__init__.py

```diff
@@ -1,6 +1,6 @@
 from .checkoutScript import checkoutTemplate
-from .configureScriptYAML import yamlInfo
-from .configureScriptXML import convert
+from .configure_script_yaml import yamlInfo
+from .configure_script_xml import convert
```

Comment:
Just curious, was this a cosmetic improvement or necessary? I prefer snakes or camels myself so agree with this and more.

Reply:
"cosmetic", implying, "please, pylint, stop yelling at me".

but, as a matter of taste, and convention, mixed/camel case in python is reserved for certain contexts, i.e. classes. perhaps these blocks of code were slated for promotion to a class, @bcc2761 ?

Reply:
@ilaflott @ceblanton Nothing significant, just that my Java brain was wired to use CamelCase for some reason and didn't realize Python doesn't usually do the same